PukingPredator/Assets/Scripts/Puking.cs

```diff
@@ -75,8 +80,38 @@ private void GameInput_Puke()
             //...the velocity and if there was a rigid body then we can reduce the velocity
             //...while calculating it based on the mass?
             itemRb.AddForce(pukeVelocity * itemRb.mass, ForceMode.Impulse);
+
+            // -1 because player loses an item when they puke
+            if (pukeForce > MAX_PUKE_FORCE * PUKE_EXPLODE_THRESH)
```

Comment:
"gameInput.pukeHoldDuration / MAX_PUKE_DURATION > PUKE_EXPLODE_THRESH" is a more direct condition for what youre looking for (unless we decide in the future that the force shouldn't scale linearly and that this should still be tied to it, but the behaviour should still be reasonable)

Reply:
forgot to do min(1, ...) on that value. its fine to keep it either way i guess, internally it'll do the same thing either way